01-node-tutorial/answers/customEmitter.js

```diff
@@ -0,0 +1,19 @@
+const EventEmitter = require("events");
+
+const customEmitter = new EventEmitter();
+
+let studentAge = 21;
+
+setInterval(() => {
+    customEmitter.emit('meeting',  'Anna', 'New York', studentAge)
+}, 1000);
+
+customEmitter.on('meeting', (name, location, age)=>{
+    console.log(`This is ${name}. She is ${age} years old and she is from ${location}.`)
+    studentAge++;
```

Comment:
This increments the variable from line 5. I think you meant `age++`. Same for the usage on the next line.